health-services/plan-service/src/main/resources/db/migration/main/V20240923113045__plan_facility_create_ddl.sql

```diff
@@ -0,0 +1,17 @@
+-- Table: plan_facility_linkage
+CREATE TABLE plan_facility_linkage (
+  id varchar(64),
+  tenant_id varchar(64),
+  plan_configuration_id varchar(64),
+  facility_id varchar(64),
+  residing_boundary varchar(64),
+  service_boundaries varchar(2048),
+  additional_details JSONB,
+  active boolean,
+  created_by varchar(64),
+  created_time bigint,
+  last_modified_by varchar(64),
+  last_modified_time bigint,
```

Comment:
since the project relies on handling epoch timestamps in milliseconds, we'll continue using bigint for consistency with the existing code and external systems."